src/main/java/br/com/excaladashboard/models/Cliente.java

```diff
@@ -16,20 +14,76 @@ public class Cliente {
     @Column(name = "id")
     private Long id;
 
+    @Column(name = "ativo", length = 256)
+    private String ativo;
```

Comment:
ativo tem que ser do tipo boolean (true/false)

```suggestion
    @Column(name = "ativo")
    private boolean ativo;
```